ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
I believe there is a yet-to-be-implemented CA side portion which will allow us to continue to sign TLS certs with the old root key until the new root cert is rolled out to the whole cluster. I'm not entirely clear on that part.

Presumably the part of this PR that loads the new root CA into the security config will be need to be changed at that point, so that managers renewing their keys don't overwrite the special transitional rootCA object that I'm positing above.